libs/community/langchain_community/document_loaders/notiondb.py

```diff
@@ -42,6 +46,7 @@ def __init__(
             "Notion-Version": "2022-06-28",
         }
         self.request_timeout_sec = request_timeout_sec
+        self.filter_object = filter_object
```

Comment:
`{}` is a mutable, convention is to typically use a `None` as the default value

Common pattern to treat with the None assignment above

```python
self.filter_object = filter_object or {}
```

Reply:
We need to implementer `filter_object or {}` otherwise self.filter_object will be assigned None by default which presumably will not work with the API, same for the case below